crypto/aes_gcm.go

```diff
@@ -0,0 +1,167 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha512"
+	"encoding/json"
+	"fmt"
+	"hash"
+
+	"github.com/bndw/pick/errors"
+	"golang.org/x/crypto/pbkdf2"
+)
+
+type AESGCMClient struct {
```

Comment:
Are you asking if we should export the `AESGCMSettings` and `AESGCMStore` structs, or  the `config` and `settings` fields of `AESGCMClient`?